VB-Application/src/main/resources/lang/GUI.properties

```diff
@@ -55,7 +55,7 @@ label.enabled_extensions=Enabled Extensions (for current project)
 label.events=Events
 label.installed_extensions=Installed Extensions
 label.other=Other
-label.plugin_author=Author:
+label.plugin_author=Author(s):
```

Comment:
https://github.com/OfficialDonut/VisualBukkit/blob/5f891442b066fe850b42c9f18b2be22cfc27ed20/VB-Application/src/main/java/com/gmail/visualbukkit/project/PluginBuilder.java#L232
This has to be changed to support multiple authors

Reply:
Oh, yeah, sure

Reply:
done